issue#9306.patch

```diff
@@ -0,0 +1,590 @@
+diff --git a/CHANGELOG.md b/CHANGELOG.md
+index 6e5a785389..b7136d7689 100644
+--- a/CHANGELOG.md
++++ b/CHANGELOG.md
```

Comment:
Pleaes delete this file or at least do not commmit it

Reply:
Sorry, this is my oversight, I will delete this file!

Reply:
Do you mean delete the entire patch file? Where should I submit a patch file if I want to? I really appreciate your help!